ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -8,113 +8,181 @@
       },
     },
     {
       id: 'VarisEx Electrified Gunshield',
       regex: Regexes.ability({ source: 'Varis Yae Galvus', id: '4CD7', capture: false }),
+      regexDe: Regexes.ability({ id: '4CD7', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.ability({ id: '4CD7', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.ability({ id: '4CD7', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
       delaySeconds: 21,
       response: Responses.knockback(),
     },
     {
       id: 'VarisEx Reinforced Gunshield',
       regex: Regexes.ability({ source: 'Varis Yae Galvus', id: '4CD9', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4CD9', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4CD9', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4CD9', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
       delaySeconds: function(data) {
         if (data.phase == 'phase2')
           return 20;
         return 10;
       },
       alertText: {
         en: 'Stop attacking',
+        ja: 'ブロックしない側に攻撃',
+        cn: '攻击未格挡的方向',
       },
     },
     {
       id: 'VarisEx Loaded Gunshield',
       regex: Regexes.ability({ source: 'Varis Yae Galvus', id: '4CD8', capture: false }),
+      regexDe: Regexes.ability({ id: '4CD8', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.ability({ id: '4CD8', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.ability({ id: '4CD8', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
       delaySeconds: 10,
-      infoText: {
-        en: 'Spread soon',
-      },
+      response: Responses.spread(),
     },
     {
       id: 'VarisEx Reinforcements',
       regex: Regexes.ability({ source: 'Varis Yae Galvus', id: '4CEA', capture: false }),
+      regexDe: Regexes.ability({ id: '4CEA', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.ability({ id: '4CEA', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.ability({ id: '4CEA', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
       infoText: function(data) {
         if (data.role == 'tank') {
           return {
             en: 'Grab Tethers',
+            ja: '線を取る',
+            cn: '接线',
           };
         }
         return {
-          en: 'Kill Adds',
+          en: 'Kill adds',
+          de: 'Adds besiegen',
+          fr: 'Tuez les adds',
+          ja: '雑魚を処理',
+          ko: '쫄 잡기',
+          cn: '击杀小怪',
         };
       },
     },
     {
       id: 'VarisEx Terminus Est Clones',
       regex: Regexes.startsUsing({ source: 'Terminus Est', id: '4CB4', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4E4F', source: 'Terminus Est', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4E4F', source: 'Terminus Est', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4E4F', source: 'ターミナス・エスト', capture: false }),
```

Comment:
I think you accidentally put the wrong ability ID in the localized versions here.

Reply:
Oh yes, I forgot to change the id when I copied from other code blocks. I will fix it soon.